commerce_coordinator/apps/titan/tasks.py

```diff
@@ -102,8 +101,22 @@ def payment_processed_save_task(
             payment_state_paid_cache_key = get_paid_payment_state_cache_key(payment_number)
             TieredCache.set_all_tiers(payment_state_paid_cache_key, payment, settings.DEFAULT_TIMEOUT)
         elif payment_state == PaymentState.FAILED.value:
+            if 'metadata' in provider_response_body:
+                if 'payment_number' in provider_response_body['metadata']:
+                    provider_response_body['metadata']['payment_number'] = ''
+                if 'order_number' in provider_response_body['metadata']:
+                    order_uuid = provider_response_body['metadata']['order_number']
```

Comment:
why we are getting order_uuid here from `provider_response_body`? `order_uuid` is already available here that was extracted from Stripe event metadata.

Reply:
removed overlooked that it was already coming from the stripe event